docs/release-notes/latest.md

```diff
@@ -72,10 +72,11 @@ tags:
 ### Added
 
 <!-- Dependency graphs? -->
-- [Dependency Paths](/semgrep-supply-chain/dependency-search#view-the-dependency-path) are now available for the following languages and package managers:
-  - **JavaScript**: all package managers are supported by Semgrep.
+- [Dependency Paths](/semgrep-supply-chain/dependency-search#view-the-dependency-path) are now available in **private beta** for the following languages and package managers:
+  - **JavaScript**: npm, pnpm, and Yarn are supported.
   - **Python**: Only Poetry is supported.
-- **C#**: Semgrep can now scan NuGet codebases without the need for a lockfile. This feature is in **private beta**. See also [Scan a project without lockfiles](/semgrep-supply-chain/getting-started#scan-a-project-without-lockfiles-beta). Reach out to [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com) to join the beta program.
+  Reach out to [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com) to join the beta program.
+- **C#**: Semgrep can now scan NuGet codebases without the need for a lockfile. This feature is in **public beta**. See also [Scan a project without lockfiles](/semgrep-supply-chain/getting-started#scan-a-project-without-lockfiles-beta). Reach out to [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com) to join the beta program.
```

Comment:
@s-santillan asked recently what it means for this to be public beta if it requires reaching out, which I think is a good question, so maybe worth confirming here as well.